packages/backend/src/modules/attachment/attachment.controller.ts

```diff
@@ -0,0 +1,45 @@
+import express from 'express'
+import { AttachmentDTO, CreateAttachmentDTO } from 'shared'
+import { Attachment } from './attachment'
+import { AttachmentService, attachmentService } from './attachment.service'
+import { AttachmentMapper, attachmentMapper } from './attachment.mapper'
+
+class AttachmentsController {
+  private readonly _attachmentMapper: AttachmentMapper
+  private readonly _attachmentService: AttachmentService
+
+  constructor(attachmentService: AttachmentService, attachmentMapper: AttachmentMapper) {
+    this._attachmentMapper = attachmentMapper
+    this._attachmentService = attachmentService
+  }
+
+  async createAttachment(req: express.Request, res: express.Response) {
+    const createdAttachmentDto: CreateAttachmentDTO = { taskId: req.body.taskId, file: req.file }
+
+    const savedAttachment: Attachment = await this._attachmentService.createAttachment(
+      createdAttachmentDto
+    )
+    const savedAttachmentDto: AttachmentDTO = this._attachmentMapper.mapToDto(savedAttachment)
+    return res.status(201).json(savedAttachmentDto)
+  }
+
+  async findById(req: express.Request, res: express.Response) {
```

Comment:
use try catch to handle errors. Right now when the attachementId is wrong the whole app is crashing. Thats definetly what we want do avoid. 404 is ok for error like this